### PR TITLE
Feature/add and remove

### PR DESCRIPTION
**Pull request by mugberto**
# Project: To-Do list/ add&remove items
In this project,
- We implemented a function for adding a new task.
- We implemented a function for editing task descriptions.
- We implemented a function for deleting a task.
- We implemented a function for the "Clear all completed" button.

### CODE REVIEW

File: src/form.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    this.taskList = taskList;
+    this.input = document.createElement('input');
+    this.input.type = 'text';
+    this.input.className = 'edit-input';
+    this.trash = document.createElement('button');
+    this.trash.type = 'button';
+    this.trash.innerHTML = '<i class="fa fa-trash"></i>';
+    this.trash.className = 'rightmost trash';
+  }
+
```
Comment (carlos-ssh) [inline src/form.js:12]: - Kindly install the [correct linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for your proyect.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mugberto , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @mugberto, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by mugberto** [comment]
Thank you @carlos-ssh for showing me where to find those linters. I took the ones in the Javascript folder, I wasn't aware of the folder called html-css-js

**Pull request merged**